notebooks/406-3D-pose-estimation-webcam/406-3D-pose-estimation.ipynb

```diff
@@ -0,0 +1,597 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9de9a93e-9247-4799-a5bb-2ec1575ae8c2",
+   "metadata": {},
+   "source": [
+    "# Live 3D Human Pose Estimation with OpenVINO\n",
+    "\n",
+    "This notebook demonstrates a live 3D Human Pose Estimation webcam example with OpenVINO. We utilize the model [human-pose-estimation-3d-0001](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/human-pose-estimation-3d-0001) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/). At the last session of this notebook, you will see the live inference results from your webcam (if available). Alternatively, you can also upload a video file to test out the algorithms.\n",
+    "**Make sure you have properly installed the [Jupyter extension](https://github.com/jupyter-widgets/pythreejs#jupyterlab) and using Jupyterlab to run the demo as mentioned in the README.txt**\n",
+    "\n",
+    "> **NOTE**: _To use the webcam, you must run this Jupyter notebook on a computer with a webcam. If you run on a server, the webcam will not work. However, you can still do inference on a video in the final step. This demo utilizes the python interface of Three.js integrated with webgl to process data derived from model inference. And it displays the processed result in the notebook._\n",
+    "\n",
+    "_To ensure that the results are displayed correctly, we recommend that you run the code in a browser on the following operating system:_  \n",
+    "_Ubuntu, Windows: chrome_  \n",
+    "_MacOS: Safari_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a9332fb-1cee-4faa-9555-731ddf0e0df7",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "316ad889-8514-430f-baf4-4f32abd43356",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import collections\n",
+    "import sys\n",
+    "import time\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import cv2\n",
+    "import ipywidgets as widgets\n",
+    "import numpy as np\n",
+    "from IPython.display import clear_output, display\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils\n",
+    "\n",
+    "sys.path.append(\"./modules\")\n",
+    "import modules.engine3js as engine\n",
+    "from modules.parse_poses import parse_poses"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1fce3d8c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# use visualization library\n",
+    "engine3D = engine.Engine3js(grid=True, axis=True, view_width=400, view_height=300)"
```

Comment:
This object should be created just before the first usage (before run_pose_estimation function)